Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Detect_Service_README.md

```diff
@@ -28,9 +36,12 @@ This playbook does not use any commands.
 | ScanNumber | Scan number in case there are multiple IDs in the parent playbook. |  | Required |
 
 ## Playbook Outputs
+
 ---
 There are no outputs for this playbook.
 
 ## Playbook Image
+
 ---
-![Cortex ASM - Detect Service](../doc_files/Cortex_ASM_-_Detect_Service.png)
\ No newline at end of file
+
+![Cortex ASM - Detect Service](../doc_files/Cortex_ASM_-_Detect_Service.png)
```

Comment:
Not sure if its a git thing, I can see a new line at the end of the file when I pull the changes locally, but I don't see it here.

Reply:
I think it is stripped out from one of the CICD jobs, but not sure.  Either way, this doesn't seem to be a blocker, right?